changelog.d/6572.bugfix

```diff
@@ -0,0 +1 @@
+Change displayname of users as admin in rooms.
```

Comment:
could you add some more words here? remember that the changelog is supposed to explain to Synapse admins what has changed: this really doesn't. Also feel free to add an attribution to yourself.

Something like this?

> When a user's profile is updated via the admin API, also generate a displayname/avatar update for that user in each room. Contributed by @dklimpel.

Reply:
Thanks for your suggestion and review.